curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples.english.md

```diff
@@ -18,13 +18,25 @@ More resources:
 
 ```yml
 question:
-  text: 'Which does the same thing as the following code?:
-<pre>lst = []<br>for key, val in counts.items():<br>    newtup = (val, key)<br>    lst.append(newtup)<br><br>lst = sorted(lst, reverse=True)<br>print(lst)<pre>'
+  text: |
+    Which does the same thing as the following code?:
+    ```python
+    lst = []
+    for key, val in counts.items():
+        newtup = (val, key)
+        lst.append(newtup)
+    lst = sorted(lst, reverse=True)
+    print(lst)
+    ```
   answers:
-    - 'print( sorted( [ (k,v) for k,v in counts.items() ] ) )'
-    - 'print( [ (k,v) for k,v in counts.items().sorted() ] )'
-    - 'print( sorted( [ (k,v) for k,v in counts.keys() ] ) )'
-    - 'print( [ (k,v) for k,v in counts.values().sort() ] )'
+    - |
+        print( sorted( [ (k,v) for k,v in counts.items() ] ) )
```

Comment:
```suggestion
        print( sorted( [ (v,k) for k,v in counts.items() ] ) )
```
The code in the question flips the tuple, so the answer has to as well. Also, it should look better in a ```py block, see https://github.com/freeCodeCamp/freeCodeCamp/issues/38942